### ISSUE TITLE
Suggestion: add support for Opera One

### ISSUE THREAD

**Issue by TheFr1zz** [plutonium-239/workspaces-tab-counter#1]
Hello, I wanted to count my tabs on Opera One but the workspaces tab counter only trakes individual tabs, not all tabs(picture1). My suggestion is could you expand this extension to work on opera one as well as gx?

![PicOperaOne1png](https://github.com/plutonium-239/workspaces-tab-counter/assets/143908014/3ba96626-3447-4393-bbf4-789a3c3c2062)

**Comment by plutonium-239** [comment]
Hi, thanks for notifying me. I'm looking into it

**Comment by plutonium-239** [comment]
It seems like it uses a different system for managing workspaces, (which isn't documented anywhere, I had to figure out how to access workspaces in gx mostly myself, only 1 shred of evidence)

For more peculiar behavior started by this, if you try to "inspect element" on the sidebar panel, it will a) not display anything like it's bugged b) will close the devtools window when you change workspace 
both of these don't happen on gx

tldr; this browser does not let extensions 'persist' through workspaces unlike gx

trying to find something that works

update: it's accessible through `opr.workspacesPrivate` API. again, why is there no documentation?
I'll try to fix it and update the ext soon

**Comment by TheFr1zz** [comment]
thank you so much! i wonder why they have it set up that way (to not persist through the workspaces). It seems like they would want to have plenty of documentation and ease of use to make it easier for people to make extensions and such.

**Comment by plutonium-239** [comment]
@TheFr1zz wow it has been a long time
In case you are still using opera one, I tested it on a VM and it worked great on a default install, can you also try to run it?

The coloring is a bit off in the default palette because I made the extension for GX (which has a vibrant, higher contrast palette), so will add some tweaks to the same.
